DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -20,9 +20,17 @@ L1TStage2uGT::L1TStage2uGT(const edm::ParameterSet& params):
    algoBitFirstBxInTrain_(-1),
    algoBitLastBxInTrain_(-1),
    algoNameFirstBxInTrain_(params.getUntrackedParameter<std::string>("firstBXInTrainAlgo","")),
-   algoNameLastBxInTrain_(params.getUntrackedParameter<std::string>("lastBXInTrainAlgo",""))
+   algoNameLastBxInTrain_(params.getUntrackedParameter<std::string>("lastBXInTrainAlgo","")),
+   algoBitUnpre_(-1),
```

Comment:
No need to make these two variables class members. They are only needed in the constructor and can be declared locally there.